bn_s_mp_to_decimal_fast.c

```diff
@@ -0,0 +1,234 @@
+#include "tommath_private.h"
+#include <string.h>
+#ifdef BN_S_MP_TO_DECIMAL_FAST_C
+/* LibTomMath, multiple-precision integer library -- Tom St Denis */
+/* SPDX-License-Identifier: Unlicense */
+
+/* store a bignum as a decimal ASCII string */
+mp_err s_mp_to_decimal_fast_rec(const mp_int *number, mp_int *nL, mp_int *shiftL, mp_int *mL, int precalc_array_index,
+                                int left,
+                                char **result,
+                                size_t *maxlen)
+{
+   mp_int q, nLq, r;
+   mp_err err;
+
+   if (precalc_array_index < 0) {
+      int n = mp_get_i32(number), n2 = n, t = 0, c;
+      char *i = *result;
```

Comment:
I don't care much about these things normally, but `i` is a really bad variable name.